infra/cifuzz/cifuzz.py

```diff
@@ -201,6 +211,56 @@ def run_fuzzers(fuzz_seconds, workspace):
   return True, False
 
 
+def get_lastest_build_version(project_name):
+  """Gets the latest OSS-Fuzz build version for a projects fuzzers.
+
+  Args:
+    project_name: The name of the project who's build is being retrieve.
+
+  Returns:
+    A string with the latest build version or None.
+  """
+  http_get_string = GC_STORAGE + '{0}/{0}-address-latest.version'.format(
```

Comment:
nit: use `os.path.join()` to concat these

Reply:
Done.